start_admin_sshd.sh

```diff
@@ -65,31 +67,31 @@ if trusted_user_ca_keys=$(get_user_data_keys "trusted_user_ca_keys"); then
   ((++available_auth_methods))
 fi
 
-chown "${local_user}" -R "${user_ssh_dir}"
+chown "${LOCAL_USER}" -R "${USER_SSH_DIR}"
```

Comment:
nit: flag before the positional arguments is preferred. Added `:` after the local user to discourage misusing `LOCAL_USER=ec2-user:myneatgroup`

```suggestion
chown -R "${LOCAL_USER}:" "${USER_SSH_DIR}"
```